cookiecutter/main.py

```diff
@@ -52,7 +52,8 @@ def expand_abbreviations(template, config_dict):
     return template
 
 
-def cookiecutter(template, checkout=None, no_input=False, extra_context=None):
+def cookiecutter(template, checkout=None, no_input=False,
```

Comment:
@ionelmc @hackebrot Okay, I will wait for all the other review comments, if any, and fix this along with the other issues pointed out.